pleskdistup/common/src/files.py

```diff
@@ -59,25 +59,31 @@ def get_last_lines(filename: PathType, n: int) -> typing.List[str]:
         return f.readlines()[-n:]
 
 
-def backup_file(filename: str) -> None:
+def backup_file(filename: str, ext: str = ".conversion.bak") -> None:
     if os.path.exists(filename):
-        shutil.copy(filename, filename + ".bak")
+        shutil.copy(filename, filename + ext)
 
 
-def backup_exists(filename: str) -> bool:
-    return os.path.exists(filename + ".bak")
+def backup_exists(filename: str, ext: str = ".conversion.bak") -> bool:
```

Comment:
There are several tests in [the motdtests.py](https://github.com/plesk/dist-upgrader/blob/main/pleskdistup/common/tests/motdtests.py#L9) should be fixed.
Additionally, it would be a good idea to add some tests related to backups. Up to you

Reply:
It would make it bigger, let's add test in a separate PR if it's OK?